metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala

```diff
@@ -86,6 +86,14 @@ class SemanticdbTreePrinter(
         }
       case s.MatchType(scrutinee, cases) =>
         s"${printType(scrutinee)} match { ${cases.size} cases }"
+      case s.LambdaType(scope, returnType) =>
+        val params = scope.map(_.symlinks).getOrElse(Nil)
```

Comment:
we shouldn't ignore hard links (actually, Scala3 links them by hardlinking)

```suggestion
        val params = scope.map(scopeInfo).getOrElse(Nil).map(_.displayName)
```

with

```scala
  private def scopeInfo(scope: s.Scope): List[s.SymbolInformation] =
    if (scope.symlinks.nonEmpty)
      scope.symlinks.map(symbol => s.SymbolInformation(symbol = symbol)).toList
    else
      scope.hardlinks.toList
```

Reply:
Thanks! Changed!